toonz/sources/translations/german/tnztools.ts

```diff
@@ -0,0 +1,1001 @@
+        <translation>Bleistift Modus</translation>
+    </message>
+    <message>
+        <source>Pressure</source>
+        <translation>Druck</translation>
+    </message>
+</context>
+<context>
+    <name>BrushToolOptionsBox</name>
+    <message>
+        <source>Preset Name</source>
+        <translation>Name der Vorlage</translation>
+    </message>
+    <message>
+        <source>OK</source>
+        <translation>OK</translation>
+    </message>
+    <message>
+        <source>Cancel</source>
+        <translation>Abbruch</translation>
+    </message>
+</context>
+<context>
+    <name>ControlPointEditorTool</name>
+    <message>
+        <source>Auto Select Drawing</source>
+        <translation>Zeichnung automatisch wählen</translation>
+    </message>
+</context>
+<context>
+    <name>ControlPointSelection</name>
+    <message>
+        <source>Set Linear Control Point</source>
+        <translation>Linearen Kontrollpunkt setzen</translation>
+    </message>
+    <message>
+        <source>Set Nonlinear Control Point</source>
+        <translation>Nicht-Linearen Kontrollpunkt setzen</translation>
+    </message>
+</context>
+<context>
+    <name>EditTool</name>
+    <message>
+        <source>Scale Constraint:</source>
+        <translation>Maßstab-Beschränkung</translation>
+    </message>
+    <message>
+        <source>Auto Select Column</source>
+        <translation>Spalte automatisch wählen</translation>
+    </message>
+    <message>
+        <source>Global Key</source>
+        <translation>Globale Taste</translation>
+    </message>
+    <message>
+        <source>Lock Center E/W</source>
+        <translation>Mitte [X] sperren</translation>
+    </message>
+    <message>
+        <source>Lock Center N/S</source>
+        <translation>Mitte［Y］ sperren</translation>
+    </message>
+    <message>
+        <source>Lock Position E/W</source>
+        <translation>Position ［X］ sperren</translation>
+    </message>
+    <message>
+        <source>Lock Position N/S</source>
+        <translation>Position［Y］ sperren</translation>
+    </message>
+    <message>
+        <source>Lock Rotation</source>
+        <translation>Rotation sperren</translation>
+    </message>
+    <message>
+        <source>Lock Shear H</source>
+        <translation>Scherung H sperren</translation>
+    </message>
+    <message>
+        <source>Lock Shear V</source>
+        <translation>Scherung V sperren</translation>
+    </message>
+    <message>
+        <source>Lock Scale H</source>
+        <translation>Maßstab H sperren</translation>
+    </message>
+    <message>
+        <source>Lock Scale V</source>
+        <translation>Maßstab V sperren</translation>
+    </message>
+    <message>
+        <source>Lock Global Scale</source>
+        <translation>Globalen Maßstab sperren</translation>
+    </message>
+    <message>
+        <source>E/W and N/S Positions</source>
+        <translation>E/W und N/S Positionen</translation>
```

Comment:
If you translate N/S (north/souths) and E/W (east/west) as X and Y above, you should do here to.
Or stay with the geographic direction and translate N/S to "N/S" (nord, süd) and E/W to "O/W" (ost, west).